packages/core/src/components/Table/TableHeader/TableHeader.tsx

```diff
@@ -183,6 +183,8 @@ export const HvTableHeader = forwardRef<HTMLElement, HvTableHeaderProps>(
               className={classes.sortButton}
               icon
               overrideIconColors={false}
+              tabIndex={-1}
+              aria-hidden
```

Comment:
Could we get rid of the `HvButton`?  🤔

Reply:
I left the button because we need the styling from the `HvButton` but I can use `component="div"` to remove the button role all altogether.

Reply:
Actually, we need the button behaviour as-is, because the parent `th` doesn't have any tab-stop (maybe it should?), therefore we need the `tabIndex` (therefore we need the role).

Reply:
Yup, I was looking at the problem the wrong way. I'll leave the button and just add some `aria-label` because that was the accessibility issue we were having.